vector-im/element-android
# [Device Manager] Render other sessions (PSG-668)

## Type of change


## Content


Implement other sessions section as in the new layout.

## Motivation and context
#6945 


## Screenshots / GIFs



|Other Sessions|
|-|
|![other_sessions](https://user-images.githubusercontent.com/1254401/186881362-edff18cf-a5d1-4794-a8b8-bb9be6b08eaa.png)|




## Tests



- Enable the new device management feature flag
- Go to "Security & Privacy" settings
- Press the "Show all sessions (V2, WIP)" entry
- See the other sessions are rendered as in the new layout

## Tested devices

- OS version(s):

## Checklist
